### PR TITLE
Sgi 575/dyncfg

### PR DESCRIPTION
**Pull request by chandnich**
Pull request for adding SGI-575 dynamic configuration platform support.

### CODE REVIEW

**Comment by antonio-nino-diaz-arm** [comment]
IP Review +1

**Comment by soby-mathew** [comment]
jenkins: test this please

**Comment by soby-mathew** [comment]
Please don't merge the patch yet as @npoushin has reported a problem with this PR.

File: plat/arm/css/sgi/fdts/sgi575.dts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/*
```
Comment (soby-mathew) [inline plat/arm/css/sgi/fdts/sgi575.dts]: Need confirmation that this doesn't break anything.

**Comment by npoushin** [comment]
Hi,

With the 18.04 Landing team release this causes a boot regression on arm-tf, are there any other dependencies on these patches that should be explicitly stated?

From the SGI575 model log, with these patches:

NOTICE:  BL2: v1.5(debug):v1.5-289-ge52b58d                                     
NOTICE:  BL2: Built : 15:29:55, Jun 15 2018                                     
INFO:    BL2: Doing platform setup                                              
INFO:    BL2: Loading image id 3                                                
INFO:    Loading image id=3 at address 0xff000000                               
INFO:    Image id=3 loaded: 0xff000000 - 0xff00b178                             
INFO:    BL2: Skip loading image id 23                                          
INFO:    BL2: Skip loading image id 25                                          
INFO:    BL2: Loading image id 5                                                
INFO:    Loading image id=5 at address 0xe0000000                               
INFO:    Image id=5 loaded: 0xe0000000 - 0xe00f0000                             
INFO:    BL2: Skip loading image id 27  <--------- Hangs here

File: plat/arm/css/sgi/fdts/sgi575.dts (hunk 1 above)
Comment (omasab) [reply plat/arm/css/sgi/fdts/sgi575.dts]: Nariman, hope the required changes in the build scripts to package the dtb files have been made for the landing team build. If not, could you try doing that and check if you still see this issue?

**Comment by npoushin** [comment]
There are outstanding patches on the LT scripts that need merged before this change can go in (lest it break sgi575). I am looking at those changes today and hopefully we can unblock this shortly.

**Comment by npoushin** [comment]
I can confirm that this works just fine with the landing team patches @omasab mentioned, thanks a lot for the info.

**Comment by dp-arm** [comment]
jenkins: test this please

**Pull request merged**